db/file/file.go

```diff
@@ -315,6 +315,11 @@ func (tx *Transaction) LockListContext(_ context.Context, s *schema.Schema, filt
 	return tx.LockList(s, filter, options, pg, policy)
 }
 
+func (tx *Transaction) CountContext(_ context.Context, s *schema.Schema, filter transaction.Filter) (uint64, error) {
+	_, total, err := tx.List(s, filter, nil, nil)
```

Comment:
I'm worried about performance of this method. In case of large tables, results of SELECT query may be big. Gohan will need to download all data provided by DB, load into memory and here dump it because it's not needed. I would prefer lightweight solution which will explicitly use COUNT(*), to acquire single row with single value.

Reply:
Ok I see now that's only for File DB. SQL DB is using COUNT(*).